src/api/app/controllers/webui/comments_controller.rb

```diff
@@ -32,6 +32,26 @@ def destroy
     render(partial: 'webui/comment/comment_list', locals: { commentable: @commentable }, status: status)
   end
 
+  def update
+    comment = Comment.find(params[:id])
+    authorize comment, :update?
+
+    comment.assign_attributes(permitted_params)
+    commentable = comment.commentable
```

Comment:
You can remove this line, and use the suggestion below, this variable is used only in the locals assignment, so you can use the `comment` object.